config/webpack.defaults.js

```diff
@@ -14,17 +14,46 @@ const cssOptions = {
 
 const babelOptions = {
   passPerPreset: true,
-  presets: ['babel-preset-react', 'babel-preset-env', 'babel-preset-stage-0'],
-  plugins: ['babel-plugin-transform-runtime', 'react-hot-loader/babel'],
+  presets: ['@babel/preset-react', '@babel/preset-env'],
+  plugins: [
+    '@loadable/babel-plugin',
+    [
+      '@babel/plugin-transform-runtime',
+      {
+        corejs: 2,
+      },
+    ],
+    'react-hot-loader/babel',
+    [
+      'relay',
+      {
+        artifactDirectory: 'src/__generated__',
+        schema: 'schema.graphql',
+      },
+    ],
+    '@babel/plugin-syntax-dynamic-import',
+    '@babel/plugin-syntax-import-meta',
+    '@babel/plugin-proposal-class-properties',
+    '@babel/plugin-proposal-json-strings',
+    [
+      '@babel/plugin-proposal-decorators',
+      {
+        legacy: true,
```

Comment:
Would it be possible to use this plugin in standard mode instead of legacy mode?

Reply:
It also looks like `@babel/plugin-proposal-class-properties` may need to be run in 'loose' mode if we're using legacy decorators per https://babeljs.io/docs/en/babel-plugin-proposal-decorators#note-compatibility-with-babel-plugin-proposal-class-properties